src/mcp/tools/core/get_firebase_directory.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { detectProjectRoot } from "../../../detectProjectRoot.js";
+
+export const get_firebase_directory = tool(
+  {
+    name: "get_firebase_directory",
+    description:
+      "Gets the current Firbase project directory. If this has been set using the `set_firebase_directory` tool it will return that, otherwise it will look for a PROJECT_ROOT environment variable or the current working directory of the running Firebase MCP server.",
```

Comment:
```suggestion
      "Gets the current Firebase project directory. If this has been set using the `set_firebase_directory` tool it will return that, otherwise it will look for a PROJECT_ROOT environment variable or the current working directory of the running Firebase MCP server.",
```